changelog_unreleased/api/17344.md

```diff
@@ -0,0 +1,20 @@
+#### Ignore files with `--check-ignore-pragma` (#17344 by @wnayes)
```

Comment:
The PR tile

> Ignore single files with `@noformat` / `@noprettier`

is fine as a commit title.

But it feels the feature is supported out-of-box (requires an extra flag to work, actually), so I updated to this version.